stubs/redis/redis/client.pyi

```diff
@@ -684,38 +684,297 @@ class PubSub:
+    def brpop(self, keys: Union[_Value, Iterable[_Value]], timeout: float = ...) -> Pipeline: ...
+    def brpoplpush(self, src, dst, timeout=...) -> Pipeline: ...
+    def lindex(self, name: _Key, index: int) -> Pipeline: ...
+    def linsert(
+        self, name: _Key, where: Literal["BEFORE", "AFTER", "before", "after"], refvalue: _Value, value: _Value
+    ) -> Pipeline: ...
+    def llen(self, name: _Key) -> Pipeline: ...
+    def lpop(self, name) -> Pipeline: ...
+    def lpush(self, name: _Value, *values: _Value) -> Pipeline: ...
+    def lpushx(self, name, value) -> Pipeline: ...
+    def lrange(self, name: _Key, start: int, end: int) -> Pipeline: ...
+    def lrem(self, name: _Key, count: int, value: _Value) -> Pipeline: ...
+    def lset(self, name: _Key, index: int, value: _Value) -> Pipeline: ...
+    def ltrim(self, name: _Key, start: int, end: int) -> Pipeline: ...
+    def rpop(self, name) -> Pipeline: ...
+    def rpoplpush(self, src, dst) -> Pipeline: ...
+    def rpush(self, name: _Value, *values: _Value) -> Pipeline: ...
+    def rpushx(self, name, value) -> Pipeline: ...
+    def sort(
+        self,
+        name: _Key,
+        start: Optional[int] = ...,
+        num: Optional[int] = ...,
+        by: Optional[_Key] = ...,
+        get: Optional[Union[_Key, Sequence[_Key]]] = ...,
+        desc: bool = ...,
+        alpha: bool = ...,
+        store: Optional[_Key] = ...,
+        groups: bool = ...,
+    ) -> Pipeline: ...
+    def scan(self, cursor: int = ..., match: Optional[_Key] = ..., count: Optional[int] = ...) -> Pipeline: ...
+    def scan_iter(self, match: Optional[Text] = ..., count: Optional[int] = ...) -> Iterator[Any]: ...
+    def sscan(self, name: _Key, cursor: int = ..., match: Text = ..., count: int = ...) -> Pipeline: ...
+    def sscan_iter(self, name, match=..., count=...) -> Iterator[Any]: ...
+    def hscan(self, name: _Key, cursor: int = ..., match: Text = ..., count: int = ...) -> Pipeline: ...
+    def hscan_iter(self, name, match=..., count=...) -> Iterator[Any]: ...
+    def zscan(self, name, cursor=..., match=..., count=..., score_cast_func=...) -> Pipeline: ...
+    def zscan_iter(self, name, match=..., count=..., score_cast_func=...) -> Iterator[Any]: ...
+    def sadd(self, name: _Key, *values: _Value) -> Pipeline: ...
+    def scard(self, name: _Key) -> Pipeline: ...
+    def sdiff(self, keys, *args) -> Pipeline: ...
+    def sdiffstore(self, dest, keys, *args) -> Pipeline: ...
+    def sinter(self, keys: _Key, *args: _Key) -> Pipeline: ...
+    def sinterstore(self, dest, keys, *args) -> Pipeline: ...
+    def sismember(self, name: _Key, value: _Value) -> Pipeline: ...
+    def smembers(self, name: _Key) -> Pipeline: ...
+    def smove(self, src, dst, value) -> Pipeline: ...
+    def spop(self, name, count: Optional[int] = ...) -> Pipeline: ...
+    def srandmember(self, name: _Key, number: Optional[int] = ...) -> Pipeline: ...
```

Comment:
I removed the overload here too for the same reason.

https://github.com/python/typeshed/blob/e2fd852952ff807c60819f8840e8b23664cc7593/stubs/redis/redis/client.pyi#L555-L558